README.md

```diff
@@ -19,35 +19,10 @@
 
 {% vimeo %}236358264{% endvimeo %}
 
-## How To Get Started
+## Ready to get started? 
+Check out our [Getting Started Guide](getting_started.md)
```

Comment:
given that the "Getting Started Guide" is now much more involved than the previous getting started guide, would it make sense to keep both, one on the front page as a _quick start_, then link to the "Scuttlebutt newbie guide" or something for more detail? although i can see us wanting to share more context before they leave the website, i'm concerned that people will see the "Guide" and gfto.

Reply:
yes, and no. I think there's value in a "Quick Start" guide, but i think the existing one is a bit _too_ quick. 

I'll take a stab at expanding the current one. Maybe have this section be 

```
## Ready to get started?
 
Check out the [Getting Started Guide](getting_started.md), or if you're in a rush you can use the [Quick Start Guide](quick_start.md). 
```

Also have the Getting started guide link to the quick start guide at the top for people who need it fast.

Reply:
i think the quick start guide should be on the front page, it's so quick you don't need to click a link and wait for another page to load.